database/insertData/_common/03_template_C_userRegistration.js

```diff
@@ -131,7 +152,254 @@ exports.queries = [
                         validationMessageInternal: "Password must be at least 8 characters"
                       }
                     }
-                    # TO-DO: Add Date of birth question once we have DatePicker element type
+                    {
+                      code: "PB1"
+                      index: 70
+                      title: "Page Break"
+                      elementTypePluginCode: "pageBreak"
+                      category: INFORMATION
+                    }
+                    {
+                      code: "S1PersonalInfo"
+                      index: 80
+                      title: "Section 1 - Personal information"
+                      elementTypePluginCode: "textInfo"
+                      category: INFORMATION
+                      parameters: {
+                        title: "## Personal information"
+                        style: "basic"
+                      }
+                    }
+                    {
+                      code: "Q6DOB"
+                      index: 90
+                      title: "DOB"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      parameters: { label: "Date of Birth", maxWidth: 150 }
+                      validation: {
+                        operator: "REGEX"
+                        children: [
+                          {
+                            operator: "objectProperties"
+                            children: ["responses.thisResponse"]
+                          }
+                          "^([0]?[1-9]|[1|2][0-9]|[3][0|1])[./-]([0]?[1-9]|[1][0-2])[./-]([0-9]{4}|[0-9]{2})$"
+                        ]
+                      }
+                      validationMessage: "Format expected DD/MM/YYYY"
+                      # TODO: Update to be using a DatePicker element type
+                    }
+                    {
+                      code: "Q7NationalID"
+                      index: 90
+                      title: "National ID number"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      parameters: { label: "National ID number" }
+                      validation: {
+                        operator: "REGEX"
+                        children: [
+                          {
+                            operator: "objectProperties"
+                            children: ["responses.thisResponse"]
+                          }
+                          "^[0-9()-]+$"
```

Comment:
Yes, I think I was basing on documents that have just numbers... This will be one of the things someone can refine later in case it's needed. For now just an example of a validation for this field.